### PR TITLE
Update latest API for component auctions explainer

### PR DESCRIPTION
**Pull request by jasarora-google**

### CODE REVIEW

File: bidding_auction_services_multi_seller_auctions.md [hunk 1]
```diff
@@ -430,12 +417,88 @@ would pass both the top-level contextual ad winner and the encrypted Protected A
...
+##### SelectAdResponse
+The SelectAdResponse for Server component Auctions will contain the following additional fields:
+
+* _string key_id_: Version of the public key used for encryption by component seller SFE. The top-level seller SFE needs to use private keys corresponding to same key_id to decrypt 'component_auctions_results'.
+
+```
+syntax = "proto3"; 
+
+ message SelectAdResponse {
+   // Existing fields
```
Comment (chatterjee-priyanka) [inline bidding_auction_services_multi_seller_auctions.md:462]: Can you please fix the indentation here?

File: bidding_auction_services_multi_seller_auctions.md [hunk 2]
```diff
@@ -491,7 +563,8 @@ message GetComponentAuctionCiphertextsRequest {
 // Returns encrypted Protected Audience data for each seller. The ciphertext
 // for each seller is generated such that they are unique. 
 message GetComponentAuctionCiphertextsResponse {
-	// Map of sellers passed in request to their encrypted ProtectedAudienceInput.
+  // Map of sellers passed in request to their encrypted ProtectedAudienceInput.
+  // Key ID is baked into the payload as done on device
```
Comment (chatterjee-priyanka) [inline bidding_auction_services_multi_seller_auctions.md:567]: End with a period.

File: bidding_auction_services_multi_seller_auctions.md [hunk 3]
```diff
@@ -240,78 +240,65 @@ followed by top level auction on the browser.
...
-
- // ProtectedAudienceInput is generated and encrypted by the client, 
- // passed through the untrusted seller's ad service, and decrypted by 
- // the SellerFrontEnd service.
- // It is the wrapper for all of BuyerInput and other information 
- // required for the Protected Audience auction.
- message ProtectedAudienceInput {
-   // Existing fields .... 
+* _string top_level_seller(optional)_: It’s similar to the seller field passed in the single seller auctions and will signal the 
+  Bidding and Auction services that this is a Component auction. This is passed in the deviceSignals field to the generateBid and
```
Comment (chatterjee-priyanka) [inline bidding_auction_services_multi_seller_auctions.md:247]: nit: generateBid() and reportResult() code by ad-techs.

**Comment by chatterjee-priyanka** [comment]
Thanks @jasarora-google for sending the PR and updating the explainer. Left a few comments, overall looks good. Will approve after you address. Thanks.

**Comment by jasarora-google** [comment]
Thanks @chatterjee-priyanka. Addressed PR comments. Please take a look.

**Review by chatterjee-priyanka — APPROVED** [review]

**Pull request merged**